components/clarity-repl/src/repl/interpreter.rs

```diff
@@ -33,6 +33,31 @@ use super::remote_data::HttpClient;
 use super::settings::{ApiUrl, RemoteNetworkInfo};
 use super::{ClarityContract, DEFAULT_EPOCH};
 
+#[derive(Debug, Clone)]
+pub enum ContractCallError {
+    NoSuchContract(String),
+    NoSuchFunction(String),
+    Uncategorized(String),
+}
+
+impl std::fmt::Display for ContractCallError {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        match self {
+            ContractCallError::NoSuchContract(contract_id) => {
+                write!(f, "NoSuchContract({})", contract_id)
```

Comment:
Clippy 1.88 is going to make you inline these

```suggestion
                write!(f, "NoSuchContract({contract_id})")
```